.github/workflows/ci-cd-gradle.yml

```diff
@@ -38,6 +37,12 @@ jobs :
       environment: preprod
       service-name: ${{ inputs.service-name}}
 
+  clear-outdated-runs:
+    needs: deploy-preprod
+    uses: ./.github/workflows/clear-runs.yml
+    secrets:
+      github-token: ${{ secrets.machine-token }}
```

Comment:
The PR is draft as it will need the machine user reviewer and secrets set up (in different repos?) before it is merged.

Reply:
Changed `secrets.machine-token`  into `secrets.MACHINE_TOKEN`. As the required machine user PAT would be the same among all the repo. Consider using secret directly from this (.github) repo rather than passing it from the calling repos, so we don't need to set the secrets and update the workflows file on every single repos.